code/game/objects/structures/barsign.dm

```diff
@@ -89,22 +94,15 @@
 				set_sign(new /datum/barsign/hiddensigns/empbarsign)
 			panel_open = 0
 
-	if(istype(I, /obj/item/stack/cable_coil) && panel_open)
-		var/obj/item/stack/cable_coil/C = I
-		if(emagged) //Emagged, not broken by EMP
-			to_chat(user, "<span class='warning'>Sign has been damaged beyond repair!</span>")
-			return
-		else if(!broken)
-			to_chat(user, "<span class='warning'>This sign is functioning properly!</span>")
-			return
 
-		if(C.use(2))
-			to_chat(user, "<span class='notice'>You replace the burnt wiring.</span>")
-			broken = 0
-		else
-			to_chat(user, "<span class='warning'>You need at least two lengths of cable!</span>")
-	else
-		return ..()
+/obj/structure/sign/barsign/attack_hand(mob/user as mob)
```

Comment:
```suggestion
/obj/structure/sign/barsign/attack_hand(mob/user)
```
`as mob` does nothing here. It's also the old way of doing things. Nowadays just a `mob/user` is plenty